server.ts

```diff
@@ -23,15 +24,15 @@ config.servers.forEach(server => {
 
         if (!matches) { console.log(key); return text; }
 
-        [,assetName,,extension] = matches;
+        [,assetName,assetFingerprint,extension] = matches;
```

Comment:
I think this is wrong? It uses the `assetFingerprint` from the URL, you want to use the one on disk?